rabbitmq/json-exchanges.yml

```diff
@@ -0,0 +1,123 @@
+type: json
+data:
+  url: /api/exchanges
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: exchanges
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: exchanges.message_stats.ack
```

Comment:
Instead of `message_stats` just use `messages` (please do the same in other metrics). Also add `.count` suffix to make it clear what it is about. So: `exchanges.messages.ack.count`.